datasets/InputData_Mixed.csv

```diff
@@ -0,0 +1,7 @@
+timestamp,Outside Air Temperature (C),Outside Air Relative Humidity (1-100%),Outside Air Humidity Ratio (-),Zone Thermostat Requested Sensible Heat Transfer Rate (kW),Supply Air Temperature (C),Supply Air Relative Humidity (0-100%),Supply Air Humidity Ratio,Water Use (Liters/min),System Electric Power (kW),Operating Mode,Supply Air Mass Flow Rate (kg/s),Supply Air Volume Flow Rate (m3/s),Sensible System Cooling Capacity (kW),Latent System Cooling Capacity (kW),Total System Cooling Capacity (kW)
```

Comment:
I'm a little confused what this file is used for.  It just has data for a few random August timestamps?  Doesn't seem like an actual "dataset", so if it needs to tag along with the idf example files, it should also go along into testfiles/ and with a more descriptive, specific filename.

Reply:
Sorry that was no longer supposed to be in the branch, I removed it.